dss/api/subscriptions.py

```diff
@@ -3,35 +3,174 @@
 import json
 import re
 import typing
+from uuid import uuid4
 
 import iso8601
 import requests
 
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import NotFoundError
+from elasticsearch_dsl import Search
 from flask import jsonify, make_response, redirect, request
 from werkzeug.exceptions import BadRequest
 
-from .. import get_logger
-from .. import DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE, DSS_ELASTICSEARCH_QUERY_TYPE
-from .. import DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME, DSS_ELASTICSEARCH_SUBSCRIPTION_TYPE
+from .. import (get_logger,
```

Comment:
If you want to make your code extra reviewable (particularly in the future), you can put a NL after your `(` and everything in subsequent lines will be indented +4, rather than be subject to the contents of the previous line.

Just a comment. Not a requirement.